src/components/map/rent-popup.tsx

```diff
@@ -87,6 +87,15 @@ const RentPopup: React.FC<RentPopupProps> = ({
       });
   }, [booking, fetchBooking, fetchStationDetail, selectedStation, BUCHUNGEN]);
 
+  const handleRefreshBooking = useCallback(() =>
+    refreshBooking()
+      .then(() => Promise.all([fetchBooking(), fetchStationDetail()]))
```

Comment:
Please see current master how I've changed the other hooks to implement the discussed data consistency measures.